src/js/program/program-controller.js

```diff
@@ -78,7 +78,7 @@ class ProgramController extends Events {
         // Reset the mediaModel now to synchronously "cancel" play promise callbacks
         // that check `mediaModel === model.mediaModel`
         if (mediaController) {
-            mediaController.off();
+            mediaController.attached = false;
```

Comment:
Just remove off, and don't use `attached`. We can put it back and add listeners earlier if we need to.